src/main/java/com/laytonsmith/core/functions/DataHandling.java

```diff
@@ -2815,25 +2780,22 @@ public String getName() {
 
 		@Override
 		public Integer[] numArgs() {
-			return new Integer[]{Integer.MAX_VALUE};
+			return new Integer[]{2};
 		}
 
 		@Override
 		public String docs() {
-			return "void {ivar | key, value} Stores a value in the global storage register."
-					+ " When using the first mode, the ivariable is stored so it can be imported"
-					+ " later, and when using the second mode, an arbitrary value is stored with"
-					+ " the give key, and can be retreived using the secode mode of import. The first mode will"
-					+ " be deprecated in future versions, so should be avoided. If"
-					+ " the value is already stored, it is overwritten. See {{function|import}} and"
-					+ " [[CommandHelper/import-export|importing/exporting]]. The reference to the value"
-					+ " is stored, not a copy of the value, so in the case of arrays, manipulating the"
-					+ " contents of the array will manipulate the stored value. export() is threadsafe.";
+			return "void {key, value} Stores a value in the global storage register."
+					+ " An arbitrary value is stored with the given key, and can be retreived using import."
+					+ " If the value is already stored, it is overwritten. See {{function|import}}."
+					+ " The reference to the value is stored, not a copy of the value, so in the case of"
+					+ " arrays, manipulating the contents of the array will manipulate the stored value."
+					+ " export() is threadsafe.";
```

Comment:
We should add a blurb about the array('key1', 'key2') format in the docs, as well as probably add an example. This is true for both functions.